pkg/pillar/Dockerfile.in

```diff
@@ -40,14 +40,16 @@ FROM WATCHDOG_TAG as watchdog
 FROM RKT_TAG as rkt-build
 # hadolint ignore=DL3006
 FROM RKT_STAGE1_TAG as rkt-stage1-build
+# hadolint ignore=DL3006
+FROM FSCRYPT_TAG as fscrypt-build
 
 FROM alpine:3.8
 RUN apk add --no-cache \
     yajl xz bash openssl iptables ip6tables iproute2 dhcpcd \
     apk-cron coreutils dmidecode sudo libbz2 libuuid ipset \
     libaio logrotate pixman glib curl radvd perl ethtool \
     util-linux e2fsprogs libcrypto1.0 xorriso \
-    python libpcap libffi jq
+    python libpcap libffi jq linux-pam
```

Comment:
I am a little bit worried about linux-pam into pillar image. What happens if we don't add this package?

Reply:
Hi @rvs fscrypt has a runtime dependency on libpam.so, which comes from this package.

Reply:
https://github.com/google/fscrypt/blob/master/README.md
(Please check "Runtime dependencies" section )

Reply:
I understand that it does -- but what exactly does it depend on and can that dependency be isolated?

I'm worried on two accounts:
   1. it increases the size of the pillar container (which I'd rather avoid)
   2. it actually is a dangerous dependency in how it plumbs the rest of Linux userland (this may not be as much true in Alpine, but it definitely true in Ubuntu)

Reply:
@rvs Actually I've reworked this to get rid of PAM dependency. Please take a look now.  
The reason fscrypt needs libpam.so is for automatic management of locking/unlocking
 of encrypted directories based on user login/logout events. Since that is not applicable to 
EVE(i.e. we don't have multi-user mode), I have put in a patch to get rid of those code 
segments, thus removing the need to have libpam.so as well.